sw/device/silicon_creator/rom/data/rom_e2e_testplan.hjson

```diff
@@ -6,6 +6,18 @@
   name: "rom_e2e"
 
   testpoints: [
+    {
```

Comment:
nit: I don't think we need this testpoint in this file since there are other test points that cover this.

Reply:
So we wanted a test point that just returns true so it runs as fast as possible, yet uses ROM to to a quick check things are working in DV. It is already covered by other tests obviously since any ROM E2E test that succeeds would do the same, but the idea is this would be fast enough to run as a presubmit test in private CI. Do you have a suggestion for another file to place this in? Maybe the chip_testplan.hjson?

Reply:
merging this but will submit a follow up PR if we decide to move this around.